roles/dosomething.sixpack/templates/sixpack.yaml.j2

```diff
@@ -0,0 +1,14 @@
+redis_port: 6379                            # Redis port
+redis_host: localhost                       # Redis host
+redis_prefix: sixpack                       # all Redis keys will be prefixed with this
+redis_db: 15                                # DB number in redis
```

Comment:
I took this config straight from the sixpack docs - however in general separating out the databases by ID (you can't use name) means you can separately drop the database or backup or whatever rather than key scanning on prefix.